python/tchannel/tornado/connection.py

```diff
@@ -554,4 +574,19 @@ def send_request(self, request):
         future = tornado.gen.Future()
         self._outstanding[request.id] = future
         self.post_request(request)
+
+        def adapt_tracing(f):
+            # fetch the request tracing for response
+            f.result().tracing = request.tracing
```

Comment:
I don't see any difference between this and create a new result. Also I am appending tracing result which is None.